wordpress-mobile/WordPress-Android
# Bump min sdk version to 26

## Description

This PR bumps min sdk version to 26 and addresses lint errors, that raised after the bump. Internal reference: p1720698865814969-slack-C04PWEZSYFL

## To Test:


This PR shouldn't change behavior changes. To test, make sure that on both WordPress and Jetpack apps:

- icons of the app look good
  - for Jetpack, verify "Themed icons" feature in both dark and light modes. Details: #19643
- shortcuts (long press on the app icon) icons look as expected too
- smoke test some vital features
